Python/Product/Analysis/LanguageServer/Messages.cs

```diff
@@ -334,6 +335,25 @@ public struct RenameParams {
         public int? _version;
     }
 
+    [Serializable]
+    public class PythonAnalysisExtensionParams {
+        public string assembly;
```

Comment:
readonly props?

Reply:
None of these have readonly props - they're mostly for serialization.